types/onfleet__node-onfleet/Resources/Tasks.d.ts

```diff
@@ -164,6 +163,23 @@ declare namespace Task {
     eta: number;
     trackingViewed: boolean;
   }
+
+  interface WorkerTaskContainer {
+    type: 'WORKER';
+    worker: string;
+  }
+
+  interface OrganizationTaskContainer {
+    type: 'ORGANIZATION';
+    organization: string;
+  }
+  
```

Comment:
@JMalaval the linter is failing due to a trailing whitespace
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/50771/checks#step:5:106
```suggestion
 
```